modules/core/src/lib/layer.js

```diff
@@ -488,7 +488,7 @@ export default class Layer extends Component {
 
     // calculateInstancePickingColors always generates the same sequence.
     // pickingColorCache saves the largest generated sequence for reuse
-    const cacheSize = pickingColorCache.length / 3;
+    const cacheSize = Math.floor(pickingColorCache.length / 3);
```

Comment:
You want it to be an integer? Would `Math.ciel` be safer?

Reply:
Math.floor works well here. With ceil we can end up assigning to non-existing cells in the typed array below.